packages/app/obojobo-express/migrations/20180124212301-create-visits-table.js

```diff
@@ -39,11 +39,9 @@ exports.up = function(db) {
 			},
 			is_preview: { type: 'boolean', notNull: true }
 		})
```

Comment:
The skipped index changes need to be added in a new migration.

Can't change the past when it comes to migrations, though we should omit the ones that never run from this migration).

Reply:
Woops. You right. I'll open a separate issue for the migration and omitting the non-running code here, since it's unrelated to the scope of this PR.